tests/common/common_by_value.h

```diff
@@ -0,0 +1,242 @@
+/*******************************************************************************
+//
+//  SYCL 1.2.1 Conformance Test Suite
+//
+//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
```

Comment:
I do not know how to update this when it comes from a file which is later improved by someone else like here...
IANAL @Ruyk

Reply:
Since this is new file created I removed this notice for now. Please let me know if any other action is required.